06-building-a-program-start-to-finish/solving-the-problem/Output/inverted-index.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Inverted Index class
+ */
+class InvertedIndex {
+    /**
+     * class constructor
+     */
+  constructor() {          
```

Comment:
Try to be consistent with your indentation across the application, keeping in mind DocBlocks, comments and methods.